### PR TITLE
Skip slow integration tests

### PR DESCRIPTION
**Pull request by not522**
## Motivation

Unintentionally, integration tests marked as `slow` are running in CI for PR. This bug was introduced in #4820.

## Description of the changes
This PR makes CI skip slow integration tests.

### CODE REVIEW

File: .github/workflows/tests-integration.yml
Comment (c-bata) [inline .github/workflows/tests-integration.yml:78]: Should we run "slow" tests on scheduled CI? Otherwise, we may not be able to detect when these slow tests fail.

Comment (not522) [reply .github/workflows/tests-integration.yml:78]: Yes, we run `slow` and `not slow` tests when we specify `-m "integration"`.

**Review by HideakiImamura — APPROVED** [review]
LGTM.

**Comment by HideakiImamura** [comment]
@c-bata Please merge this PR after your approval.

**Comment by HideakiImamura** [comment]
@gen740 Could you review this PR?

**Review by gen740 — APPROVED** [review]
LGTM!

**Pull request merged**
